ScienceJournal/UI/EditExperimentViewController.swift

```diff
@@ -229,25 +229,24 @@ class EditExperimentViewController: MaterialHeaderViewController, EditExperiment
 
   // MARK: - ImageSelectorDelegate
 
-  func imageSelectorDidCreateImageData(_ imageData: Data, metadata: NSDictionary?) {
+  func imageSelectorDidCreateImageData(_ imageDatas: [(imageData: Data, metadata: NSDictionary?)]) {
     // Check if the photo can be saved before proceeding. Normally this happens lower in the stack
     // but we do it here because it is currently not possible to pass the error up the UI to here.
-    if metadataManager.canSave(imageData) {
-      guard let image = UIImage(data: imageData) else { return }
+    guard imageDatas.count == 1, let imageDataTuple = imageDatas.first else {
+      fatalError("Only one image can be selected for the edit experiment vc.")
```

Comment:
A thought... instead of a fatalError, why not instead just take [0] from the array?

Reply:
I was thinking about that too, I just figured, if somehow we managed to get more than 1 image when we were expecting only 1, it might be better to catch this programming error/assumption error. I could go either way, though.

Reply:
Actually, I'll do what you suggested and add some logging for this condition since it's not really worthy of a fatal